server/data/dbConfig.js

```diff
@@ -1,6 +1,5 @@
 const knex = require('knex');
 const config = require('../knexfile.js');
-
-const dbEnv = 'staging';
+const dbEnv = process.env.DB_ENV || 'development';
```

Comment:
This will overwrite let it as we will need it latter. Make sure to accept and keep your changes not mine.